pages/en/hosted-service/deploy-subgraph-hosted.mdx

```diff
@@ -52,16 +52,14 @@ In some cases, you will want to deploy the same subgraph to multiple Ethereum ne
 
 ### graph-cli >=0.29.0
```

Comment:
Should we change this heading since the options were only added in `v0.32.0` for `graph deploy`?

Reply:
I wasn't sure about that either, because the options are available in graph-cli for the `build` command since v0.29.0, we just made them available for `deploy` too, and now you can use whichever.

Reply:
Or I can change the heading to `Using graph-cli` and `Using templates/templating` or something of that sort

Reply:
> Or I can change the heading to Using graph-cli and Using templates/templating or something of that sort

I think that would be better! 👍